gordian/repo.py

```diff
@@ -57,7 +62,8 @@ def get_objects(self, filename, klass=None):
 
     def get_files(self):
         if not self.files:
-            contents = self._repo.get_contents("")
+            logger.debug(f'Getting repo content')
+            contents = self._repo.get_contents('/')
```

Comment:
Is `get_contents('/')` the same as `get_contents('')`?

Reply:
oops I didn't want to change that

Reply:
It is, I checked the library implementation, when the argument is an empty string it internally uses '/', but I just reverted it anyways